### DISCUSSION TITLE
Win7 image lightening

### DISCUSSION THREAD

**Discussion by johdcmlaker2000** [tetrane/reven-community#14] [Q&A] [answered]
In addition to the steps described in the image deployment documentation, is it necessary to use ntlite and your windows lightening script to speed up the windows 7 x64 image?

Will it have a significant impact on performance and reduce the size of the replay?

**Comment by dureuill** [comment, marked as answer]
Hi,

The NTLite template provided by Tetrane, as well as the Windows lightening script, have been specifically designed to lighten Windows 10 images. The result of their use on W7 is unspecified.

However, while [configuring a W7 guest](https://doc.tetrane.com/free/latest/Quick-Start/Import-VM/Windows-7.html#configuring-the-guest), you still need to insert the Windows 10 lightener CDROM in order to e.g. disable the KPTI protections.

Hope this helps!
